inst/htmlwidgets/shotsign.js

```diff
@@ -52,7 +52,7 @@ HTMLWidgets.widget({
     // y = Field Goal % in shooting signatures
     var y = d3.scale.linear()
       .domain(xx.ydomain ? xx.ydomain : [0, 1])
-      .range([height, 0 + maxWidth]);
+      .range([height, 0 + margin.top]);
```

Comment:
@timelyportfolio I'm not entirely sure why this was set to be `maxWidth` in the original code, but it results in just having a lot of extra space at the top of most of my signatures which is kind of annoying.  Do you have any issues with this?